chart/templates/api/serviceaccount.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.api.serviceAccount.create -}}
+apiVersion: v1
+kind: ServiceAccount
```

Comment:
just for my personal information, what's the use of a service account here? (i've only seen it used for controllers or prometheus probes)